lte/gateway/c/oai/tasks/s1ap/s1ap_state_converter.h

```diff
@@ -52,6 +52,20 @@ class S1apStateConverter : StateConverter {
   static void s1ap_imsi_map_to_proto(
       const s1ap_imsi_map_t* s1ap_imsi_map, oai::S1apImsiMap* s1ap_imsi_proto);
 
+  /**
```

Comment:
As a convention, we add the serialize and deserialize functions together so it is easier to read and debug. Most often a change to serialization logic will require a change to deserialization logic.